schema_based_on_diagram.sql

```diff
@@ -0,0 +1,54 @@
+-- Create patient table
+DROP TABLE IF EXISTS patients;
+CREATE TABLE patients(
+	id INT GENERATED ALWAYS AS IDENTITY PRIMARY KEY,
+	name varchar(100),
+	date_of_birth DATE
+);
+
+-- Create treatment table
+DROP TABLE IF EXISTS treatments;
+CREATE TABLE treatments(
+	id INT GENERATED ALWAYS AS IDENTITY PRIMARY KEY,
+	name varchar(100),
+	type varchar(100)
+);
+
+
+-- Create medical history table 
+DROP TABLE IF EXISTS medical_histories;
+CREATE TABLE medical_histories(
+	id INT GENERATED ALWAYS AS IDENTITY PRIMARY KEY,
+	admitted_at TIMESTAMP,
+	patient_id INT REFERENCES patients(id),
+	status varchar(100)
+);
```

Comment:
- [ ] It's a requirement that all "join tables" for the many-to-many relationships are created, even if they are not shown on the diagram. Therefore, you should create a table between medical_histories and treatments